packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -657,10 +678,21 @@ export default class New extends Component {
                       </div>
                     </fieldset>,
                     <fieldset>
-                      <label>{_('newSrUseNfs4')}</label>
-                      <div>
-                        <Toggle onChange={this.toggleState('nfs4')} />
-                      </div>
+                      <label htmlFor='selectSrPath'>{_('newSrNfs')}</label>
+                      <select
```

Comment:
`htmlFor='selectSrPath'` ?  
You need `id` property in `select` if you use `htmlFor`.